### PR TITLE
Small fixes and 'm_' prefix removal

### PR DESCRIPTION
**Pull request by DanVanAtta**
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/586)

### CODE REVIEW

File: src/games/strategy/util/IntegerMap.java [hunk 1]
```diff
@@ -46,40 +46,36 @@ public IntegerMap(final Collection<T> objects, final int value) {
...
   public IntegerMap(final IntegerMap<T> integerMap) {
-    /*
-     * this will also work:
-     * m_values = new HashMap<T,Integer>(integerMap.m_values);
-     */
-    m_values = new HashMap<T, Integer>(integerMap.size());
+    this.mapValues = new HashMap<T, Integer>(integerMap.size());
     for (final T t : integerMap.keySet()) {
-      m_values.put(t, integerMap.getInt(t));
+      this.mapValues.put(t, integerMap.getInt(t));
```
Comment (ron-murhammer) [inline src/games/strategy/util/IntegerMap.java]: Can we remove the 'this' here?

**Comment by ron-murhammer** [comment]
@DanVanAtta 1 comment added.

File: src/games/strategy/util/IntegerMap.java (hunk 1 above)
Comment (DanVanAtta) [inline src/games/strategy/util/IntegerMap.java]: Unnecessary 'this' keywords do not bother me. If I had to argue about it, I would suggest it's probably better to have them than not. Not a huge deal either way IMO, I added a commit to remove the unnecessary 'this'

Comment (ron-murhammer) [inline src/games/strategy/util/IntegerMap.java]: Yeah, not a big deal just prefer consistency. We seem to currently be leaning towards only using 'this' when necessary.

**Pull request merged**